docs/bar-series.md

```diff
@@ -35,6 +35,14 @@ Type (VerticalBarSeries): `number`
 Type (HorizontalBarSeries): `string|number`
 The y position in coordinates of the box to be used. For VerticalBarSeries this value is considered a number, and is scaled against it's domain into pixels.
 
+<!-- INJECT:"DifferenceChartWithLink" -->
+
+#### y0
+(Optional)
+Type (VerticalBarSeries): `number`  
+Type (HorizontalBarSeries): `string|number`
+The y0 position in coordinates of the box to be used, this is where the bottom of the bar is placed, defaults to zero. Use is not recommended with stacked bars. For VerticalBarSeries this value is considered a number, and is scaled against it's domain into pixels.
```

Comment:
This was a very useful tidbit.  I assume if one wanted to do DifferenceHorizontalBar, one sets x and x0 in a similar manner?

Reply:
Yep that's just right!